l1-contracts/test/unit_tests/transaction_filterer.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { expect } from "chai";
```

Comment:
While additional tests are always appreciated, considering that we'll migrate from hardhat to foundry soon, the hardhat tests will be eventually deleted. Is there something they test that the foundry tests don't?

Reply:
I think it's just the difference in ownership tests for coverage, but I could do the same in forge, so no real benefit, so deleting it